### PR TITLE
Updated content for getting started docs section

### PR DESCRIPTION
**Pull request by bmicklea**
Content updates for the pages inside the Getting Started section of the Codenvy 5.0 docs.

### CODE REVIEW

**Comment by bmicklea** [comment]
Note that I'm suggesting we delete the `environments` page in the Getting Started section since it appears to duplicate content in `user guide` > `sharing`.

File: docs/_docs/getting-started/architecture.md [hunk 1]
```diff
@@ -4,4 +4,6 @@ excerpt: ""
 layout: docs
 permalink: /docs/architecture/
 ---
+Codenvy is composed of 8 Docker containers in a compose relationship. A browser is required for administration of the Codenvy system and to use the (optional) Eclipse Che browser IDE.
```
Comment (JamesDrummond) [inline docs/_docs/getting-started/architecture.md]: Do we need to specify what kind of browsers can be used or link?

File: docs/_docs/getting-started/getting-started.md [hunk 2]
```diff
@@ -4,15 +4,24 @@ excerpt: ""
...
-| **Developer** who will be creating custom extensions, plug-ins, and assemblies to package Codenvy and Eclipse Che into new tools. | [*The Developer Guide*](../../docs/rest-api). These documents provide all of our API interfaces, both internal and external. You will need to get comfortable with using the Eclipse Che SDK, and there is extensive additional SDK documentation that will make you an innovation beast. Also, if you are going to place your extensions into a Codenvy On-Prem, check out the additional [Plug-Ins](../../docs/developing-extensions/) administrator steps. |
+| **Developer** who will be creating custom extensions, plug-ins, and assemblies to package Codenvy and Eclipse Che into new tools. | [*Che Plugin Guide*](https://eclipse-che.readme.io/docs/introduction-1). These docs will help you setup your dev evironment and walk you through how to craete server- and client-side extensions and plugins. If you are going to place your extensions into a Codenvy On-Prem instance please [contact us for guidance](https://codenvy.com/contact/questions/). |
```
Comment (JamesDrummond) [inline docs/_docs/getting-started/getting-started.md]: craete -> create . 

`[*Che Plugin Guide*](https://eclipse-che.readme.io/docs/introduction-1)`. This is a good example of why having all docs available on codenvy site. Take a look at https://codenvy.github.io/5_0_0-M8/docs/introduction-1/ . Even though it's not found in the navigation on right you can still view the page. Cross linking to che.github.io would be difficult I think.

So I think we should change `https://eclipse-che.readme.io/docs/introduction-1` to `../../docs/introduction`

File: docs/_docs/getting-started/introduction.md [hunk 3]
```diff
@@ -4,10 +4,11 @@ excerpt: "Welcome to Codenvy!"
 layout: docs
 permalink: /docs/introduction/
 ---
-Codenvy makes on-demand workspaces to give you a better agile experience. Codenvy SaaS is available for self-service at codenvy.com. You can also install Codenvy on your own infrastructure with Codenvy On-Prem.
+Codenvy makes Cloud workspaces for development teams featuring one-click Docker environments, team onboarding and a workspace platform
+for DevOps. Codenvy SaaS is available for self-service at codenvy.io. You can also install Codenvy on your own infrastructure with Codenvy On-Prem.
```
Comment (JamesDrummond) [inline docs/_docs/getting-started/introduction.md]: Was the line break intended between 7-8?

File: docs/_docs/getting-started/getting-started.md (hunk 2 above)
Comment (bmicklea) [inline docs/_docs/getting-started/getting-started.md]: I'll fix the typo.

Although it's cool that you can see the page, it's confusing not to have it in the TOC and I can't jump to the other pages in that section so jumping away to the Che docs site is better. We can update the link after with the correct github.io link.

File: docs/_docs/getting-started/introduction.md (hunk 3 above)
Comment (bmicklea) [inline docs/_docs/getting-started/introduction.md]: no, i'll fix.

**Comment by bmicklea** [comment]
@JamesDrummond - updated typos.

File: docs/_docs/getting-started/getting-started.md (hunk 2 above)
Comment (JamesDrummond) [inline docs/_docs/getting-started/getting-started.md]: Yeah it is confusing. We might consider adding this section to codenvy TOC. For now I think your edits are good and we can change later if we decide to do so.

**Pull request merged**